src/services/git/model.ts

```diff
@@ -35,7 +35,7 @@ export interface Commit {
   url: string;
   author: Author;
   message: string;
-  tree: Tree;
+  tree: Tree | Tree[];
```

Comment:
why?

Reply:
It seems repo commits and pull commits return the same "parents" as an array.

https://api.bitbucket.org/2.0/repositories/pypy/pypy/commits/
https://api.bitbucket.org/2.0/repositories/pypy/pypy/pullrequests/93/commits

Reply:
just select the first parent from the list